### DISCUSSION TITLE
!mp ban user timer and unban command

### DISCUSSION THREAD

**Discussion by HeyItzRaiderZ** [ppy/osu#21610] [Ideas] [answered]
Just a suggestion unless already talked about.

**Comment by smoogipoo** [comment, marked as answer]
Firstly, this is an osu-stable request and should go in https://github.com/ppy/osu-stable-issues.

Secondly, I don't understand what this would do. Prevent players from joining matches? I don't agree with this and this is not the intended use of multiplayer. Make the lobby private/add a password.
